app/src/main/java/org/sil/gatherwords/SessionActivity.java

```diff
@@ -1,127 +1,133 @@
 
-    private FusedLocationProviderClient mFusedLocationClient;
-    boolean locationEnabled;
-    Location location;
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_session);
-
-        locationEnabled = false;
-
-        long date = System.currentTimeMillis() / 1000L;
-
-        EditText dateField = findViewById(R.id.session_create_date);
-        dateField.setText(Long.toString(date));
-        dateField.setEnabled(false);
-
-
-    }
-
-    public void save_settings_fab_pressed(View view) {
-        EditText name = findViewById(R.id.session_create_name);
-        EditText eliciter = findViewById(R.id.session_create_eliciter);
-        EditText speaker = findViewById(R.id.session_create_speaker);
-        EditText date = findViewById(R.id.session_create_date);
-
-        Session session = new Session();
-        session.setData("name", name.getText().toString());
-        session.setData("elictier", eliciter.getText().toString());
-        session.setData("speaker", speaker.getText().toString());
-        session.setData("date", date.getText().toString());
-
-        Intent i = new Intent(this, MainActivity.class);
-        startActivity(i);
-    }
-
-    public void input_location(View view) {
-        if (locationEnabled) {
-            locationEnabled = false;
-            location = null;
-        } else {
-            mFusedLocationClient = LocationServices.getFusedLocationProviderClient(this);
-            location = new Location("vanDellen 362");
-            locationEnabled = false;
-
-            if (ActivityCompat.checkSelfPermission(this, Manifest.permission.ACCESS_FINE_LOCATION) != PackageManager.PERMISSION_GRANTED) {
-                ActivityCompat.requestPermissions(this,
-                        new String[]{Manifest.permission.ACCESS_FINE_LOCATION},
-                        1);
-            } else {
-                locationEnabled = true;
-            }
-
-            if (locationEnabled) {
-                setSessionLocation();
-            }
-        }
-    }
-
-    private void setSessionLocation() {
-        LocationManager mLocationManager = (LocationManager) getSystemService(Context.LOCATION_SERVICE);
-
-        if (!mLocationManager.isProviderEnabled(LocationManager.GPS_PROVIDER)) {
-            SwitchCompat sw = findViewById(R.id.session_create_location_swtich);
-            Snackbar mySnackbar = Snackbar.make(findViewById(R.id.session_create_layout),
-                    "Please enable location services to access this feature", Snackbar.LENGTH_LONG);
-            mySnackbar.show();
-            sw.setChecked(false);
-            locationEnabled = false;
-        } else {
-            location = mLocationManager.getLastKnownLocation(LocationManager.GPS_PROVIDER);
-            //TODO: Something with this location
+	private FusedLocationProviderClient mFusedLocationClient;
```

Comment:
Why have you converted spaces to tabs?  Whitespace changes hide actual functional changes from the diff.  Also, they make tracing via `git blame` challenging.  (Most of the rest of the codebase is also already using spaces.)

Reply:
I've always used tabs, and my Studio is set up that way and has been for a while.

Reply:
Could you switch your setup to spaces for this project?  Otherwise each time you commit, it will rewrite the entire file, making review and merging challenging.

Reply:
done